tools/cldr-apps/js/src/views/ReportResponse.vue

```diff
@@ -60,7 +67,15 @@ export default {
       return cldrReport.reportName(this.report);
     },
     statusClass() {
-      return cldrReport.reportClass(this.completed, this.acceptable);
+      return `d-dr-${this.reportStatus.status}`;
```

Comment:
The CSS styles were designed to be constructed this way, from the beginning… The suffix is vote resolver status

Reply:
We could put a comment into the CSS… Also, locality of reference can help, if we split up the translated strings so that a section of them had to do with this specific view